src/node/rheeprom.cpp

```diff
@@ -2,6 +2,8 @@
 #include "config.h"
 #include "rheeprom.h"
 
+#if !STM32_MODE_FLAG

```

Comment:
Is this a good time to drop eeprom, as discussed in #349 ?

Reply:
Not looking to increase the scope of this PR. :]

If we're moving on to S32_BPill boards then it may not be worth the trouble. One place where the EEPROM is useful is if the Arduino node is reset because of I2C issues.

Reply:
Agreed. We should address later, but no need to hold up the PR.